cdap-api/src/main/java/co/cask/cdap/api/app/AbstractApplication.java

```diff
@@ -225,24 +227,25 @@ protected void addService(String name, HttpServiceHandler handler, HttpServiceHa
    * @param schedule the schedule to be added for the Workflow
    * @param workflowName the name of the Workflow
    */
+  @Deprecated
```

Comment:
The `Scheduler` currently works on `Schedule` object. But in fact, it works on `TimeSchedule` isn't it? My question is how common are things between `TimeSchedule` and `StreamSizeSchedule`?